src/services/remoteService.ts

```diff
@@ -326,6 +328,30 @@ export class RemoteService extends Disposable implements IRemoteService {
         throw new Error('SSH password modal dialog, Canceled');
     }
 
+    async updateRemoteSSHConfig(usingSSHGateway: boolean, localAppSSHConfigPath: string | undefined) {
```

Comment:
is it only move, without any semantic changes?

Reply:
yep